src/components/MovieCard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import '../styles/MovieCard.scss'
+
+const MovieCard = props => {
+  const splitDate = props.releaseDate.split('-')
```

Comment:
Really good learning on how to reformat the date.  I will be remembering this for future projects